### PR TITLE
Add quickstart content for Actions and Hooks

### LINKED ISSUE #6082: Add hooks example to quickstart

### PR DESCRIPTION
**Pull request by rmoff**
- Remove existing action config and scripts
- Add actions&hooks page to quickstart, tidy up hooks docs too

 

### Linked Issue

Closes #6082

---

### Background

Actions are a really cool feature of lakeFS, and we should introduce users to them as part of the quickstart. 

### New Feature

Quickstart page covering use of simple Lua hook & action

### CODE REVIEW

**Comment by rmoff** [comment]
Needs two icons creating before we can merge it, but ready for someone to try out and review.

File: docs/quickstart/actions-and-hooks.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+
+We're going to enforce a rule that when a commit is made to any branch that begins with `etl`: 
+
+* the commit message must not be blank
+* there must be `job_name` and `version` metadata
+* the `version` must be numeric
+
+To do this we'll create an _action_. In lakeFS, an action specifies one or more events that will trigger it, and references one or more _hooks_ to run when triggered. Actions are YAML files written to lakeFS under the `_lakefs_actions/` folder of the lakeFS repository.
+
+_Hooks_ can be either a [Lua](/hooks/lua.html) script that lakeFS will execute itself, an external [web hook](/hooks/webhook.html), or an [Airflow DAG](/hooks/airflow.html). In the example here we're using a Lua hook.
```
Comment (eladlachmi) [inline docs/quickstart/actions-and-hooks.md:23]: Something about the current wording sounds off to my ear, but maybe it's just me :)
```diff
- In the example here we're using a Lua hook.
+ In this example, we'll use a Lua hook.
```

**Review by eladlachmi — APPROVED** [review]
Very minor non-blocking comment. Aside from that, LGTM.

File: docs/quickstart/actions-and-hooks.md (hunk 1 above)
Comment (rmoff) [reply docs/quickstart/actions-and-hooks.md]: thanks - fixed.

**Pull request merged**